packages/@sanity/schema/src/sanity/extractSchema.ts

```diff
@@ -180,11 +180,6 @@ export function extractSchema(
       return {type: 'unknown'} satisfies UnknownTypeNode // we don't support cross-dataset references at the moment
     }
 
-    // Global document references are not supported
-    if (isGlobalDocumentReferenceType(schemaType)) {
```

Comment:
This will need a fix on the Create side (both backend and frontend) so we ignore gdr fields there for now.
Ie, we would have to get that out first before this can be merged.
If left as is, Create form will render this as an object with whatever fields GDR type has.